single_iwyu.py

```diff
@@ -99,8 +99,8 @@ def perform_iwyu(fixer_path: Path, part: json, filters: List[Path], current_path
         warn("HEADER POTENTIALLY MODIFIED")
 
     command[-2] = str(preprocess_file)
-    subprocess.check_call(command + ['-E'])
     try:
+        subprocess.check_call(command + ['-E'])
         new_size = linecount(preprocess_file)
```

Comment:
can the rest of the statements that cannot raise that except be brought outside of the try block? after it